bundles/org.openhab.binding.miio/src/main/resources/ESH-INF/thing/vacuumThing.xml

```diff
@@ -127,6 +129,16 @@
 		<category>Energy</category>
 		<state pattern="%.0f" readOnly="true" />
 	</channel-type>
+	<channel-type id="water_box_status">
+		<item-type>Switch</item-type>
+		<label>Water Box State</label>
+		<state readOnly="true" />
```

Comment:
This should be changeable with the `set_water_box_custom_mode` right? Or is that yet another field?
If it is this one, suggest not to mark it as readonly, but instead add the command as well

Reply:
No, this is not related to the `set_water_box_custom_mode`. It only indicates whether the mob module is installed or not. It is a simple status flag that is read only.

Reply:
But isn't that more a property than a channel? This is not expected to change, is it?

Reply:
Can you comment on this one?